simulation/amaru-sim/src/simulator/generate.rs

```diff
@@ -307,6 +309,28 @@ pub fn generate_inputs_strategy(
     })
 }
 
+fn exponential<R: Rng>(rng: &mut R, mean: f64) -> f64 {
```

Comment:
I think the [rand_distr](https://docs.rs/rand_distr/latest/rand_distr/) crate provides standard distributions, perhaps we could use that?